website/releases/release-0.15.0.md

```diff
@@ -0,0 +1,339 @@
+
+#### New and Deprecated Flink Bundles
+
+We have expanded Hudi support to Flink 1.18 with a new
+bundle, [hudi-flink1.18-bundle](https://mvnrepository.com/artifact/org.apache.hudi/hudi-flink1.18-bundle). This release
+removes Hudi support on Flink 1.13.
+
+### Module and API Changes
+
+#### Hudi Storage and I/O Abstractions
+
+This release introduces new storage and I/O abstractions that are Hadoop-agnostic to improve integration with query
+engines, including [Trino](https://trino.io/), which uses its own native File System APIs. Core Hudi classes
+including [`HoodieTableMetaClient`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java),
+[`HoodieBaseFile`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/model/HoodieBaseFile.java),
+[`HoodieLogFile`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/model/HoodieLogFile.java),
+[`HoodieEngineContext`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/engine/HoodieEngineContext.java),
+etc., now depend on new
+storage and I/O classes. If you’re using these classes directly in your applications, you need to change your
+integration code and usage. For more details, check out [this section](#hudi-storage-and-io-abstractions-1).
+
+#### Module Changes
+
+As part of introducing new storage and I/O abstractions and making core reader logic Hadoop-agnostic, this release
+restructures the Hudi modules to clearly reflect the layering. Specifically,
+
+- [`hudi-io` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-io) is added to for I/O related
+  functionality, and the Hudi-native HFile reader implementation sits inside this new module;
+- [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) contains the core
+  implementation of the [Apache Hudi Technical Specification](https://hudi.apache.org/tech-specs) and is now
+  Hadoop-independent;
+- [`hudi-hadoop-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-hadoop-common) contains
+  implementation based on Hadoop file system APIs to be used
+  with [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) on engines
+  including Spark,
+  Flink, Hive, and Presto.
+
+If you use [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) as the dependency
+before and Hadoop file system APIs and implementations, you should include
+all three
+modules, [`hudi-io`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-io), [`hudi-common`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common),
+and [`hudi-hadoop-common`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-hadoop-common), as the dependency
+now.
```

Comment:
should we add a line here stating for presto and trino a new version will be released based on 0.15.0 w/ approriate changes. Or if we already have a version available, can we link it here.

Reply:
Added.